### PR TITLE
- eslint - lint file .eslintrc.js

### PR DESCRIPTION
**Pull request by kaizhu256**
this is the first of several pull-requests to tighten javascript-styleguide for sql.js.

the final pull-request will be to enforce linting on /test/ and /examples/ directory.

the following eslint-rule was tightened:
```diff
-"comma-dangle": "off",
+"comma-dangle": ["error", "never"],
```

the following file was added to eslint:
- `/.eslintrc.js`

added npm-command `prettify` to autofix minor whitespace/indentation style-violations:
```shell
npm run prettify
```

### CODE REVIEW

**Comment by lovasoa** [comment]
Why do you want to remove dangling commas ? They make it easier to add new elements to arrays and parameters to functions, and make commit diffs clearer.
The only argument that I could find against them is that they don't work in IE8, but this doesn't apply to us since our code then passes through a minifier that removes them.

**Comment by lovasoa** [comment]
However, a pull request standardizing our tests to apply the same eslint rules to them as to the code would be welcome !

**Comment by kaizhu256** [comment]
ok relaxed eslint-rule back to `"comma-dangle": "off"` in follow-on commit 88c832e2.  are there any other issues?

would still like this pull-request merged, because its:

1. small and easy-to-review
2. less controversial and less discussions-required than follow-on pull-request for /tests/.
3. the code-changes in this pull-request are required anyways in order to eslint /tests/.

File: src/api.js [hunk 1]
```diff
@@ -768,7 +768,7 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
                         if (curresult === null) {
                             curresult = {
                                 columns: stmt["getColumnNames"](),
-                                values: [],
+                                values: []
```
Comment (lovasoa) [inline src/api.js:771]:
```suggestion
                                values: [],
```

**Comment by lovasoa** [comment]
> ok relaxed eslint-rule back to `"comma-dangle": "off"` in follow-on commit [88c832e]

You forgot to remove the rule

**Comment by kaizhu256** [comment]
pushed revert-commit cdd12539, sorry ^^;;;

**Pull request merged**